runtime/parachains/src/session_info.rs

```diff
@@ -37,6 +40,10 @@ pub mod migration;
 #[cfg(test)]
 mod tests;
 
+// The order of parameters should be deterministic, that is, one should not reorder them when
+// changing the array contents to avoid creating excessive pressure to PVF execution subsys.
+const EXECUTOR_PARAMS: [ExecutorParam; 0] = [];
```

Comment:
Forgive me for missing the discussion, but we start with no params at all?

Reply:
Exactly, which means "use all the defaults". Anything put here [will just overwrite the defaults](https://github.com/paritytech/polkadot/pull/6161/files#diff-f4ffe6a7003b388a382a12c2ac95731f8862f4e3cd96be2898f271c131ee528aR110-R127).

Reply:
That makes sense. Maybe it's worth adding a comment to the default constants that they are never meant to be modified in the code and the way to change them is a runtime upgrade with documenting steps?

Reply:
what does "excessive pressure" means here?

Reply:
> what does "excessive pressure" means here?

If one set is `[Param1(100), Param2(200)]`, and another is `[Param2(200), Param1(100)]`, then, despite the sets being the same, they produce different hashes, and execution queue should use different types of workers to execute jobs with those sets, which involves waiting times, killing workers, re-spawning workers, etc. That doesn't make much sense to overload the execution queue with that unneeded work, that's what I meant.